samples/word/99-preview-apis/manage-styles.yaml

```
@@ -0,0 +1,357 @@
+              borders.outsideBorderColor = "green";
+              console.log("Updated outside borders");
+            }
+          });
+        }
+
+        async function deleteStyle() {
+          // Deletes the custom style.
+          await Word.run(async (context) => {
+            const styleName = $("#style-name-to-delete").val() as string;
+            if (styleName == "") {
+              console.warn("Enter a style name to delete.");
+              return;
+            }
+
+            const style = context.document.getStyles().getByNameOrNullObject(styleName);
+            style.load();
+            await context.sync();
+
+            if (style.isNullObject) {
+              console.warn(`There's no existing style with the name '${styleName}'.`);
+            } else {
+              style.delete();
+              console.log(`Successfully deleted custom style '${styleName}'.`);
+            }
+          });
+        }
+
+        let externalDocument;
+
+        async function getExternalStyles() {
+          // Gets style info from another document passed in as a Base64-encoded string.
+          await Word.run(async (context) => {
+            const retrievedStyles = context.application.retrieveStylesFromBase64(externalDocument);
+            await context.sync();
+
+            console.log("Styles from the other document:");
+            console.log(retrievedStyles.value);
+          });
+        }
+
+        function getBase64() {
+          // Retrieve the file and set up an HTML FileReader element.
+          const myFile = <HTMLInputElement>document.getElementById("file");
+          const reader = new FileReader();
+
+          reader.onload = (event) => {
+            // Remove the metadata before the Base64-encoded string.
+            const startIndex = reader.result.toString().indexOf("base64,");
+            externalDocument = reader.result.toString().substr(startIndex + 7);
+          };
+
+          // Read the file as a data URL so that we can parse the Base64-encoded string.
+          reader.readAsDataURL(myFile.files[0]);
+        }
+
+        // Default helper for invoking an action and handling errors.
+        async function tryCatch(callback) {
+          try {
+            await callback();
+          } catch (error) {
+            // Note: In a production add-in, you'd want to notify the user through your add-in's UI.
+            console.error(error);
+          }
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+            This sample demonstrates how to manage styles.
+        </section>
+
+        <section class="samples ms-font-m">
+            <h3>Try it out</h3>
+            <p>
+                <h4>Get current number of styles</h4>
+                <button id="count" class="ms-Button">
+                        <span class="ms-Button-label">Get count</span>
+                </button>
+            </p>
+
+            <h4>Add a new style</h4>
```

I think "new" is implied by adding something. Adding "custom" here helps imply all the other actions are related.
```suggestion
            <h4>Add a custom style</h4>
```